home1/task1.js

```diff
@@ -0,0 +1,14 @@
+function isAnagrams(str1 = "", str2 = "") {
```

Comment:
function would return `true` for following call. Even despite those words aren't anagrams of each other
`isAnagrams("aaab", "bbba")`